app/controllers/concerns/hyrax/batch_uploads_controller_behavior.rb

```diff
@@ -42,6 +43,19 @@ def build_form
         @form.payload_concern = params[:payload_concern]
       end
 
+      def redirect_when_feature_disabled!
+        return if Flipflop.enable_batch_upload?
+        respond_to do |wants|
```

Comment:
@jcoyne I followed up on your suggestion and I'm seeing two spec failures w/ the batch_uploads_controller_spec below. I've tried a few different incantations, but the `wants` object here never seems to trigger the `json` or `html` blocks below:

```
(byebug) p wants
#<ActionController::MimeResponds::Collector:0x000000081d4108 @responses={}, @variant=[]>
```

Any ideas?